NHAPI/NHAPI/Functions.cs

```diff
@@ -160,7 +171,26 @@ private List<Guid> GetFileTransfers(Type testType, TransferType transferType)
             }
         }
 
-        
+        private string GetMethodName(string command)
+        {
+            string methodName;
+            if (!commandToMethodMap.TryGetValue(command.ToLowerInvariant(), out methodName))
+            {
+                methodName = command;
+            }
+            return methodName;
+        }
+
+        public bool NHCommand(string command, string args = "")
```

Comment:
These new methods, `NHCommand` and `NHCommand_RetrieveResults` allow easily adding support for additional NH commands. New command must be just added to the `commandToMethodMap` on line #21 dictionary together with the correspondent method name from `UI.exe` project.